src/ser/mod.rs

```diff
@@ -489,17 +503,6 @@ impl<W: fmt::Write> Serializer<W> {
         Ok(())
     }
 
-    fn indent(&mut self) -> fmt::Result {
```

Comment:
I think I'd still like to keep this method as a thin wrapper around the new function